src/components/Forms/TextField.js

```diff
@@ -254,7 +250,7 @@ class TextField extends React.Component {
             onBlur={this.handleInputBlur}
             onChange={this.handleInputChange}
             onFocus={this.handleInputFocus}
-            onKeyDown={onKeyDown && onKeyDown()}
+            onKeyDown={ (e) => onKeyDown(e) }
```

Comment:
We should break this out to `handleKeyDown` like the handlers above this line. Anonymous functions generated in the render method will register as a false prop change on render.